### PR TITLE
refactor: move module source to root

### PR DESCRIPTION
**Pull request by JustinBeckwith**
The additional subdir was unnecessary, and I suspect it's masking a few issues.

### CODE REVIEW

**Comment by stanley-cheung** [comment]
Looks like after the move, this line https://github.com/GoogleCloudPlatform/grpc-gcp-node/blob/master/grpc-gcp/test/integration/spanner_test.js#L25 needs to be fixed.

File: test/integration/spanner_test.js [hunk 1]
```diff
@@ -22,7 +22,7 @@
 
 'use strict';
 
-const PROTO_DIR = __dirname + '/../../../third_party/googleapis';
+const PROTO_DIR = __dirname + '../../third_party/googleapis';
```
Comment (stanley-cheung) [inline test/integration/spanner_test.js:25]: Looks like a leading `/` is missing

**Comment by JustinBeckwith** [comment]
@stanley-cheung the errors in the logs appear to be related to permissions....

```
Uncaught AssertionError [ERR_ASSERTION]: ifError got unwanted exception: 7 PERMISSION_DENIED: Caller is missing IAM permission spanner.sessions.create on resource projects/grpc-gcp/instances/sample/databases/benchmark.
```

Is this somehow assuming I personally have access to the gcp project?  Or did I mess something else up 😆

**Comment by stanley-cheung** [comment]
I got to here: https://github.com/GoogleCloudPlatform/grpc-gcp-node/blob/master/.kokoro/system-test.sh#L22

That's the service account credentials used for that spanner_test, but I couldn't figure out who set that `KOKORO_GFILE_DIR` environment variable.

**Comment by stanley-cheung** [comment]
@bcoe Hi, I noticed it was you who added this `system-test.sh` and added the service account credentials here: https://github.com/GoogleCloudPlatform/grpc-gcp-node/blob/master/.kokoro/system-test.sh#L22-L23. Do you have some idea about this error: `IAM permission spanner.sessions.create on resource projects/grpc-gcp/instances/sample/databases/benchmark`?

**Comment by stanley-cheung** [comment]
On the other hand, #82 seems to be the last PR who has this "ci/kokoro: System test" passed, but the PR #82 was merged on about 10 months ago 2020/07/07. I couldn't find any other successful run after that point.

**Comment by stanley-cheung** [comment]
@bcoe Gentle ping. We are having some permission issues here with the kokoro system test you introduced a while ago. Thanks!

**Comment by bcoe** [comment]
@stanley-cheung are you able to grant permissions on `projects/grpc-gcp/instances/sample/databases/benchmark` for the service account, 

`kokoro-system-test@long-door-651.iam.gserviceaccount.com`

I believe at some point these must have been revoked, which appears to be leading to the failures.

**Comment by stanley-cheung** [comment]
@bcoe I don't think I have permission in the `long-door-651` project. After I selected that project and try to click on Spanner, I got a permission error.

**Comment by bcoe** [comment]
@stanley-cheung I believe the permission needs to be granted on the project `grpc-gcp`, which is what unit tests appear to be talking to.

I'm not sure who created this testing project?

**Comment by stanley-cheung** [comment]
I checked with @WeiranFang and we both don't have access to the project `grpc-gcp` now.

I am still a little bit confused about this `long-door-651` project here though: https://github.com/GoogleCloudPlatform/grpc-gcp-node/blob/master/.kokoro/system-test.sh#L23

So where do we go from here?

**Comment by stanley-cheung** [comment]
@JustinBeckwith Perhaps you can try if you can give me Owner access to this `google-cloud-node` project?

**Comment by bcoe** [comment]
@stanley-cheung the reason I'm fixating on the project `grpc-gcp` is that the errors I'm seeing in the log are as follows:

```
   Uncaught AssertionError [ERR_ASSERTION]: ifError got unwanted exception: 7 PERMISSION_DENIED: Caller is missing IAM permission spanner.sessions.create on resource projects/grpc-gcp/instances/sample/databases/benchmark.
```

Which suggests that the `long-door-651` service account is attempting to interact with the resource:

`projects/grpc-gcp/instances/sample/databases/benchmark`

I have access to the `long-door-651` project, but I don't know that we can get these existing tests to pass if we don't sort out its inability to access this external resource.

Could we potentially skip the tests that are the culprit?

**Comment by JustinBeckwith** [comment]
We need to get this resolved so we can safely land dependency updates and security fixes.  Instead of trying to piece together how to use multiple GCP projects, @stanley-cheung could your team create a new GCP project responsible for running CI?  Or if you like, we can figure out how to completely use our CI project - but being split between two is obviously causing issues :) @stanley-cheung what do you think?  How do we get this unstuck?

**Comment by bcoe** [comment]
Apologies for the slow reply.

@stanley-cheung let's use `long-door-651`, if that's okay with you. You have the access you need?

**Comment by nimf** [comment]
@JustinBeckwith did we replace kokoro with cloudbuild or we are going to bring back kokoro? The system-test is fixed now.

**Pull request merged**